lib/node/node.ts

```diff
@@ -83,3 +85,28 @@ if (httpClient && httpClient.ClientRequest) {
     }
   };
 }
+
+function patchNextTick() {
+  let setNative = null;
+
+  function scheduleTask(task: Task) {
+    const args = task.data;
+    args[0] = function() {
+      task.invoke.apply(this, arguments);
+    };
+    setNative.apply(process, args);
```

Comment:
> But it is allowable to use a different context, and so this could be a problem in the future. With setTimeout the DOM spec requires you to call it on an object of type Window or WorkerGlobalScope (i.e. the global scope)

I didn't know such kind of usage, thank you for the information. in this PR, I'll just keep it this way just to make the coding style be the same with patchTimer. Thanks again.